Lib/multiprocessing/process.py

```diff
@@ -296,8 +297,18 @@ def _bootstrap(self):
             traceback.print_exc()
         finally:
             util.info('process exiting with exitcode %d' % exitcode)
-            sys.stdout.flush()
-            sys.stderr.flush()
+            #check if windows platform and Python 3.4
+            if sys.platform.startswith('win') and sys.version_info[:2] == (3, 4):
+                # checks if forzen app
+                is_frozen = (hasattr(sys, "frozen") or # new py2exe
+                             hasattr(sys, "importers") or # old py2exe
+                             imp.is_frozen("__main__")) # tools/freeze
+                if not is_frozen:
+                    sys.stdout.flush()
+                    sys.stderr.flush()
```

Comment:
The fact that one frozen module crashes from the flushes does not mean that all frozen modules will or that suppressing flushes will not harm other frozen modules.  The fact that other frozen modules do run on various versions of Python suggest that the problem is with the particular frozen module.  This suggests to me that the patch is in general wrong and should be rejected now.